app/javascript/menu/menu-collapse.jsx

```diff
@@ -6,8 +6,14 @@ import { SideNavItem } from 'carbon-components-react/es/components/UIShell';
 const MenuCollapse = ({ expanded, toggle }) => (
   <SideNavItem className="menu-collapse">
     <div
+      role="button"
+      tabIndex="0"
```

Comment:
this makes tabbing from the URL go directly to collapse button, instead of miq logo